paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetViewModelTest.kt

```diff
@@ -716,7 +717,7 @@ internal class PaymentSheetViewModelTest {
 
         assertThat(viewModel.supportedPaymentMethods.size).isEqualTo(1)
         assertThat(viewModel.supportedPaymentMethods.first()).isEqualTo(
-            SupportedPaymentMethod.AfterpayClearpay
+            lpmRepository.fromCode("afterpay_clearpay")!!
```

Comment:
Can we make the assert's actual value the result of `lpmRepository.fromCode("afterpay_clearpay")!!`, e.g. `SupportedPaymentMethod.AfterpayClearpay` in this case?

Reply:
yeah, SupportedPaymentMethod.AfterpayClearpay no longer exists.  I could create a StaticLpmRepository that does have static Lpms, and then they could be used here in tests, but there is something nice about using the actual lpm json file.